server/routes.js

```diff
@@ -26,6 +26,7 @@ import graphqlSchemaV2 from './graphql/v2/schema';
 import cache from './lib/cache';
 import errors from './lib/errors';
 import logger from './lib/logger';
+import sendMessageToSupport from './lib/message-support';
```

Comment:
Maybe move this to the controller folder.

Reply:
Yes, nice, it's moved.